src/CreatorToken.sol

```diff
@@ -65,11 +65,15 @@ contract CreatorToken is ERC721 {
     string memory _name,
     string memory _symbol,
     address _creator,
+    uint256 _creatorFee,
     address _admin,
+    uint256 _adminFee,
     IERC20 _payToken
   ) ERC721(_name, _symbol) isNotAddressZero(_creator) isNotAddressZero(_admin) {
     creator = _creator;
+    CREATOR_FEE_BIPS = _creatorFee;
```

Comment:
Thinking about it now, we should probably have some bounds around the max fee that you can set. Certainly the total has to be less than 1,000 or things break. Realistically there's probably also some cap we could enforce on each fee, maybe 25%.